### PR TITLE
Add chicken button

### PR DESCRIPTION
**Pull request by CharlotteGayton**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+
 function App() {
-  
-  chickenClick = () => {
-    console.log('this is:', this);
-  };
+
+  function chickClick() {
+    multiplier = multiplier + 1;
+  }
```
Comment (danny-burrows) [inline src/App.js]: This function is currently mutating state that is local to the function `chickClick` perhaps you might want to create some shared state. See https://reactjs.org/docs/hooks-state.html for more details

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 
   return (
     <div className="App" >
       <header className="App-header">
         <h1>Farm Idle Game</h1>
-        <button type="button" onClick={chickenClick}>Click Me!</button>
+        <button type="button" onClick={chickClick}>Click Me!, {multiplier}</button>
```
Comment (danny-burrows) [inline src/App.js]: You might wish to display this counter in a separate element than the button itself?

**Review by danny-burrows — COMMENTED** [review]
Looking good, just a couple comments. <3

File: src/App.js (hunk 1 above)
Comment (CharlotteGayton) [reply src/App.js]: I have renamed them to chickRate and setChickRate and now increases by one when the button is clicked

File: src/App.js [hunk 3]
```diff
@@ -1,16 +1,23 @@
+// import { render } from '@testing-library/react';
 import './App.css';
+import React, {useState} from 'react';
+
+// const multiplier = 0;
+// const chickenRate = 0;
+// const total = 0;
```
Comment (danny-burrows) [inline src/App.js]: Why are these comments present? Are they needed?

Comment (CharlotteGayton) [reply src/App.js]: They will be removed in the next branch.

File: src/App.js [hunk 4]
```diff
@@ -1,16 +1,23 @@
+// import { render } from '@testing-library/react';
 import './App.css';
+import React, {useState} from 'react';
+
+// const multiplier = 0;
+// const chickenRate = 0;
+// const total = 0; 
+
+var start = Date.now()
```
Comment (danny-burrows) [inline src/App.js]: Why are you using the rather outdated `var` keyword as opposed to `let` or `const`?

Comment (CharlotteGayton) [reply src/App.js]: Because it works.

File: src/App.js (hunk 3 above)
Comment (danny-burrows) [reply src/App.js]: I don't like that we aren't keeping the history clean, can we just remove them now?

Comment (CharlotteGayton) [reply src/App.js]: No

File: src/App.js (hunk 4 above)
Comment (danny-burrows) [reply src/App.js]: This isn't being used though?

Comment (CharlotteGayton) [reply src/App.js]: It will be in the next branch

Comment (danny-burrows) [reply src/App.js]: But it isn't now, I just think we need to keep the history clean so this PR makes sense.

Comment (CharlotteGayton) [reply src/App.js]: I've already moved on from this branch I can't make the changes. You took so long on a call so I just had to move on

Comment (CharlotteGayton) [reply src/App.js]: ive resolved them.

File: src/App.js (hunk 3 above)
Comment (CharlotteGayton) [reply src/App.js]: ive done it. is it to your satisfaction?

Comment (danny-burrows) [reply src/App.js]: Perfect thanks!

**Comment by danny-burrows** [comment]
LGTM

**Review by danny-burrows — APPROVED** [review]
LGTM

**Pull request merged**